androidHyperskillApp/src/main/java/org/hyperskill/app/android/gamification_toolbar/view/ui/delegate/GamificationToolbarDelegate.kt

```diff
@@ -55,6 +55,14 @@ class GamificationToolbarDelegate(
                 isVisible = true
                 text = state.hypercoinsBalance.toString()
             }
+        } else {
+            // TODO: Implement skeletons for loading and idle states
+            with(viewBinding.gamificationStreakDurationTextView) {
+                isVisible = false
+            }
+            with(viewBinding.gamificationGemsCountTextView) {
+                isVisible = false
+            }
```

Comment:
Just tiny remark: it's better to use with() in case you have more than 1 `method to call`/`property to change`